CMakeLists.txt

```diff
@@ -144,23 +131,11 @@ if (DEFINED ARANGODB_VERSION_RELEASE_TYPE)
 
   # devel
   if (ARANGODB_VERSION_PATCH STREQUAL "0" AND ARANGODB_VERSION_RELEASE_TYPE STREQUAL "devel")
-    set(ARANGODB_DEBIAN_UPSTREAM "${ARANGODB_VERSION_MAJOR}.${ARANGODB_VERSION_MINOR}.0~~${ARANGODB_VERSION_PATCH}")
-    set(ARANGODB_DEBIAN_REVISION "1")
-
-    set(ARANGODB_RPM_UPSTREAM "${ARANGODB_PLAIN_VERSION}")
-    set(ARANGODB_RPM_REVISION "0.1.devel")
-
```

Comment:
Out of curiosity: Why did you delete all the variables for Debian and RPM but not for Darwin?

Reply:
This is a good question; I think I assumed that a Mac release build somehow needs this. Not sure though, will have to look/ask;